public/sass/base/_icons.scss

```diff
@@ -92,6 +92,12 @@
   background-image: url('../img/grafana_icon.svg');
 }
 
+.ft-branding {
```

Comment:
We might want to keep all of the freshtracks styling in a single file or folder. What do you think?

Reply:
yeah sounds good